samples/remote_attestation/remoteattestation.edl

```diff
@@ -18,6 +18,17 @@ enclave {
                                                [in, count=remote_report_size] uint8_t *remote_report,
                                                size_t   remote_report_size);
 
+        // Create ephemeral symmetric key, encrypt using other enclave's public key
+        // and sign using private key
+        public int establish_secure_channel(  [out] uint8_t** data,
+                                              [out] size_t*  size);
+
+        // Verify signature, decrypt using private key
+        // Send back symmetric key after encrypting using other enclave's public key
+        // and sign using private key
+        public int acknowledge_secure_channel(  [in, count=size] uint8_t* data,
+                                               size_t  size);
```

Comment:
spacing looks all messed up since size_t doesn't line up under anything

Reply:
clang-format modified this. Will try to fix this.

Reply:
ok, will fix